src/common/math_utils.hpp

```diff
@@ -157,6 +157,21 @@ template <typename T, typename A,
 inline U elu_fwd(T s, A alpha) {
     return s > 0 ? s : (U)(alpha * (::expm1f((float)s)));
 }
+
+template <typename T, typename A,
+         typename U = typename utils::remove_reference<T>::type>
+inline U swish_fwd(T s, A alpha) {
+    return (U)(s / ( 1 + ::expf(-alpha*(float)s)));
+}
+
+
+template <typename T, typename A,
+         typename U = typename utils::remove_reference<T>::type>
+inline U swish_bwd(T dd, T s, A alpha) {
+    U v = (U)(1 / (1 + ::expf((float) -s*alpha)));
+    return dd * (v + s*alpha*v*(1 - v));
```

Comment:
here and above, please add spaces around binary operations:
``` c++
s*a*v --> s * a * v
```

Reply:
fixed